res/miraclecast.spec

```diff
@@ -1,60 +1,57 @@
-%global commit c3c868e523f450ad4f0d77f5484a3b61f08120b7
+%global commit 264a222d242734da369ca287aa6cfc6ca4f1f7bf
 %global shortcommit %(c=%{commit}; echo ${c:0:7})
 
 Name:           miraclecast
 Version:        1.0
-Release:        1.git%{shortcommit}%{?dist}
-Summary:        Connect external monitors to your system via Wifi-Display
-License:        LGPL
+Release:        2.git% {shortcommit}% {?dist}
+Summary:        Connect external monitors to your system via Wi-Fi display (miracast)
+License:        LGPLv2
 URL:            https://github.com/albfan/miraclecast
 Source0:        https://github.com/albfan/miraclecast/archive/%{commit}/%{name}-%{shortcommit}.tar.gz
-BuildRequires:  autoconf, automake, libtool
-BuildRequires:  readline-devel, glib2-devel, systemd-devel
 
-%description
-The MiracleCast project provides software to connect external monitors to your system via Wi-Fi. It is compatible to the Wifi-Display specification also known as Miracast. MiracleCast implements the Display-Source as well as Display-Sink side.
+BuildRequires:  autoconf
+BuildRequires:  automake
+BuildRequires:  libtool
+
+BuildRequires:  glib2-devel
+BuildRequires:  readline-devel
+BuildRequires:  systemd-devel
 
-The Display-Source side allows you to connect external displays to your system and stream local content to the device. A lot of effort is put into making this as easy as connecting external displays via HDMI.
+%description
+The MiracleCast project provides software to connect external monitors to your
+system via Wi-Fi. It is compatible to the Wi-Fi display specification also
+known as Miracast. MiracleCast implements the Display-Source as well as
+Display-Sink side.
 
-On the other hand, the Display-Sink side allows you to create wifi-capable external displays yourself. You can use it on your embedded devices or even on full desktops to allow other systems to use your device as external display.
+The Display-Source side allows you to connect external displays to your system
+and stream local content to the device. A lot of effort is put into making
+this as easy as connecting external displays via HDMI.
 
+On the other hand, the Display-Sink side allows you to create Wi-Fi capable
+external displays yourself. You can use it on your embedded devices or even on
+full desktops to allow other systems to use your device as external display.
 
 %prep
-%autosetup -n %{name}-%{shortcommit}
-
+%autosetup -n %{name}-%{commit}
 
 %build
-mkdir build
-cd build
-../autogen.sh g --prefix=%{_prefix}
+autoreconf -fiv
+%configure
```

Comment:
do I need --enable-rely-udev ? how should I choose if I need or not ?

Reply:
add udev rules is not for everyone. So I think no